api/peer.js

```diff
@@ -134,7 +136,27 @@ export class Peer extends EventEmitter {
     this.peer = await PeerFactory.create({ ...this.opts })
 
     this.peer.onPacket = (packet, port, address, data) => {
-      this._emit(packet.usr1, packet.message, address, port)
+      const event = packet.usr1
+      let message = packet.message
+
+      if (!event || typeof event !== 'string') return
+
+      if (isBufferLike(message)) {
+        try {
+          message = textDecoder.decode(message)
+        } catch (err) {}
```

Comment:
should it emit an error here as well?

Reply:
it may not be a valid string to decode, we still want to emit the packet message buffer